scene/gui/text_edit.cpp

```diff
@@ -1229,6 +1229,66 @@ void TextEdit::backspace_at_cursor() {
 
 }
 
+void TextEdit::indent_selection_right() {
+
+	if (!is_selection_active()) {
+		return;
+	}
+	begin_complex_operation();
+		int start_line = get_selection_from_line();
```

Comment:
I agree with @neikeq that's the additional indentation here is a bit strange when compared to the rest of the code.